synapseclient/core/download/download_functions.py

```diff
@@ -161,6 +170,105 @@ async def download_file_entity(
     entity.cacheDir = os.path.dirname(download_path)
 
 
+async def download_file_entity_model(
+    *,
+    download_location: str,
+    file: "File",
+    if_collision: str,
+    submission: str,
+    synapse_client: Optional["Synapse"] = None,
+) -> None:
+    """
+    Download file entity
+
+    Arguments:
+        download_location: The download location
+        entity:           The File object
+        if_collision:      Determines how to handle file collisions.
+                            May be
+
+            - `overwrite.local`
+            - `keep.local`
+            - `keep.both`
+
+        submission:       Access associated files through a submission rather than through an entity.
```

Comment:
`synapse_client` arg is missing. Was this added in 1107?

Reply:
Nope - I'll add this now

Reply:
Added in this commit: https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1111/commits/c269dc7d830d437d15d2c382f6cc10d251ba3bb9